### ISSUE TITLE
Feature request: pre-flight check

### ISSUE THREAD

**Issue by hafriedlander** [silverstripe/cow#12]
Before starting any release process, cow should be able to check your permissions & environment to confirm you have the needed prerequisites and access.

**Comment by tractorcow** [comment]
Yep, and we should probably integrate it with @dhensby 's "automatic setup script" to install all the pre-release tools.

**Comment by robbieaverill** [comment]
cc @dnsl48 - this sounds like something you'd like =)

**Comment by dnsl48** [comment]
Yes. I'm not sure what is the automatic setup script, but we're trying to add a separate script for running cow in `release mode` that adds some extra checks (e.g. that you run the latest cow version and you have ssh access to github). I guess we can add a separate cow command for performing all those checks.
Unfortunately, I couldn't find a sane way to check `TRANSIFEX_API` access with its cli. `tx status` doesn't seem to be checking for access, it only looks for the project .tx/config correctness.
